docs/professional-services/index.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 1
+sidebar_label: Overview
+---
+
+# Professional Services Overview
+
+## Our team expertise
+
+Why you can trust our team:
+
+### Distributed systems architecture
+
+[Gauthier](https://www.linkedin.com/in/gauthier-leonard/) and [Léo](https://www.linkedin.com/in/leo-schoukroun/), our technical founders, have **years of experience operating France's biggest data clusters** based on open-source technologies.  
+They have deep knowledge of distributed systems architecture challenges and are experts in setting up highly available and scalable infrastructures.
+
+### Avalanche knowledge
+
+#### Open-source contributions
+
+Our team maintains multiple [open-source tools](/docs/toolkit/) tailored for Avalanche and is actively contributing to other projects in the Avalanche ecosystem.
+
+See examples of the team contributions:
+
+- [Ava Labs repositories](https://github.com/issues?q=is%3Aissue+involves%3ANuttymoon+involves%3Aleopaul36+archived%3Afalse+org%3Aava-labs+)
+
+#### Node operating
+
+Ash is **operating multiple Avalanche nodes** (on the mainnet and Fuji testnet), and taking part in decentralized RPC networks such as [Lava](https://lavanet.xyz).
+
+### Education
+
+[Gauthier](https://www.linkedin.com/in/gauthier-leonard/) has been a **distributed systems teacher** at some of the top French engineering schools ([CentraleSupélec](https://www.centralesupelec.fr/), [ECE Paris](https://www.ece.fr/), [DSTI](https://www.datasciencetech.institute/fr/)) for more than 3 years.
+
+## How we can help you
+
+Our team can offer you the following services:
+
+### Avalanche nodes setup and maintenance
+
+Ash [open-source tools](/docs/toolkit/) and premium [Console](/docs/console/) plans can significantly reduce the setup and maintenance cost of your Avalanche nodes.
+
+Our team can help you with the **initial setup** and provide you with a maintenance plan **tailored to your needs**.
+
+### Extra features for the Ash tools
+
+Because we are the maintainers of the Ash [open-source tools](/docs/toolkit/) and [Console](/docs/console/), we can **develop extra features** for any extra need you may have. This could be:
```

Comment:
```suggestion
As the maintainers of the Ash [open-source tools](/docs/toolkit/) and [Console](/docs/console/), we can **develop custom features** for any additional need you may have. This could be:
```

Reply:
For me "extra" is more accurate as the features would likely be open-sourced as part of the toolkit.

Reply:
How about "additional" then? Extra make it sound like they **won't** be open-sourced IMO.